SQL/database_changelog.txt

```diff
@@ -122,23 +122,6 @@ CREATE TABLE `admin_connections` (
 
 -----------------------------------------------------
 
-Version 5.14, xx May 2021, by Anturke
```

Comment:
Don't delete changelog entries. I'll add the changes for you and update the DB templates, though.